### PR TITLE
Update setup.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/setup.asciidoc [hunk 1]
```diff
@@ -18,19 +18,18 @@ The following pages will help you configure your APM Agent.
 [[configuring-the-agent]]
 === Configuring the agent
 
-There are multiple ways to configure the Node.js agent. In order of precedence (higher overwrites lower):
+Use the below methods to configure the Node.js {agent}, listed in order of prescendence (highest to lowest):
```
Comment (leahleahy) [inline docs/setup.asciidoc:21]: LMK if this rephrase makes sense. Tried to match the other articles.

**Comment by astorm** [comment]
jenkins run the tests please

File: docs/setup.asciidoc [hunk 2]
```diff
@@ -104,10 +103,10 @@ Instead you need to import the `elastic-apm-node/start` module:
 import apm from 'elastic-apm-node/start'
 ----
 
-Now, you can either configure the agent using the environment variables associated with each <<configuration,configuration option>>,
+Now, configure the {agent} using the environment variables associated with each <<configuration,configuration option>>,
 or use the optional <<agent-configuration-file,agent configuration file>>.
 
-To explicitly start the agent you need to import `elastic-apm-node` and call the `apm.start()` method with the
+To explicitly start the {agent}, import `elastic-apm-node` and call the `apm.start()` method with the
```
Comment (bmorelli25) [inline docs/setup.asciidoc:109]:
```suggestion
To explicitly start the agent, import `elastic-apm-node` and call the `apm.start()` method with the
```

File: docs/setup.asciidoc [hunk 3]
```diff
@@ -104,10 +103,10 @@ Instead you need to import the `elastic-apm-node/start` module:
 import apm from 'elastic-apm-node/start'
 ----
 
-Now, you can either configure the agent using the environment variables associated with each <<configuration,configuration option>>,
+Now, configure the {agent} using the environment variables associated with each <<configuration,configuration option>>,
```
Comment (bmorelli25) [inline docs/setup.asciidoc:106]:
```suggestion
Now, configure the agent using the environment variables associated with each <<configuration,configuration option>>,
```

File: docs/setup.asciidoc [hunk 4]
```diff
@@ -95,7 +94,7 @@ for instance with the help of Babel,
 TypeScript,
 or the `--experimental-modules` flag for Node.js,
 all import statements are evaluated prior to calling any functions.
-**In this case, you can't configure the agent by passing a configuration object to the `start` function,
+**In this case, you can't configure the {agent} by passing a configuration object to the `start` function,
```
Comment (bmorelli25) [inline docs/setup.asciidoc:97]:
```suggestion
**In this case, you can't configure the agent by passing a configuration object to the `start` function,
```

File: docs/setup.asciidoc [hunk 5]
```diff
@@ -84,8 +83,8 @@ require('elastic-apm-node').start({
 [[agent-configuration-file]]
 ==== Agent configuration file
 
-The Node.js agent will look for a file named `elastic-apm-node.js` in the current working directory.
-You can specify a custom path for this file using the <<config-file,`configFile`>> config option.
+The Node.js {agent} looks for a file named `elastic-apm-node.js` in the current working directory.
+Specify a custom path for this file using the <<config-file,`configFile`>> config option.
```
Comment (bmorelli25) [inline docs/setup.asciidoc:87]:
```suggestion
The Node.js agent looks for a file named `elastic-apm-node.js` in the current working directory.
You can specify a custom path for this file with the <<config-file,`configFile`>> configuration option.
```

File: docs/setup.asciidoc (hunk 1 above)
Comment (bmorelli25) [inline docs/setup.asciidoc:21]:
```suggestion
There are multiple ways to configure the Node.js agent. In order of precedence (higher overwrites lower):
```

File: docs/setup.asciidoc [hunk 6]
```diff
@@ -56,7 +55,7 @@ var apm = require('elastic-apm-node').start({
 })
 ----
 
-Here's an example usage configuring the agent to only be active in production:
+Below is an example of usage configuring the {agent} to only be active in production:
```
Comment (bmorelli25) [inline docs/setup.asciidoc:58]:
```suggestion
This example shows how to configure the agent to only be active in production:
```

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**